### PR TITLE
Implement development environment for a frontend part

### PR DESCRIPTION
**Pull request by xansochi**
Test pull request

### CODE REVIEW

**Review by Lehersun — CHANGES_REQUESTED** [review]
Let's remove useless files

**Comment by Lehersun** [comment]
Read it for correct commits and merge requests naming https://chris.beams.io/posts/git-commit/

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "private": false,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
```
Comment (Lehersun) [inline package.json]: We don't know what dependences do we realy need for the project, let's create fully blank project with react only

**Review by Lehersun — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (xansochi) [reply package.json]: Information on dependencies is being formed now, so I left the data empty #2

**Review by Lehersun — APPROVED** [review]

**Pull request merged**